File: BackEnd_Learn/Java/auth/src/main/java/io/github/leeseojune53/auth/exception/UserNotFoundException.java
package io.github.leeseojune53.auth.exception;

import io.github.leeseojune53.auth.error.exception.BusinessException;
import io.github.leeseojune53.auth.error.exception.ErrorCode;

public class UserNotFoundException extends BusinessException {
    public UserNotFoundException() {
        super(ErrorCode.USER_NOT_FOUND);
    }
}

File: BackEnd_Learn/kotlin/kopring/src/main/kotlin/io/github/leeseojune53/kopring/domain/user/exception/UserNotFoundException.kt
package io.github.leeseojune53.kopring.domain.user.exception

import io.github.leeseojune53.kopring.global.error.exception.ErrorCode
import io.github.leeseojune53.kopring.global.error.exception.KopringException

class UserNotFoundException private constructor() : KopringException(ErrorCode.USER_NOT_FOUND) {
	companion object{
		@JvmField
		val EXCEPTION = UserNotFoundException()
	}
}